/**:
  ros__parameters:
    target_frame: "velodyne32"
    length: [250.0, 250.0]
    cell_size: 0.1
    obstacle_separation_threshold: 1.0
    projection_dz_threshold: 0.01
    bins_numb: 0.01
    origin_z: 0.0
    robot_z: -2.5
    probability_matrix:
      occupied_to_occupied: 0.95
      occupied_to_free: 0.05
      free_to_occupied: 0.2
      free_to_free: 0.8
    v_ratio: 0.1

